### DISCUSSION TITLE
Chrome Extension

### DISCUSSION THREAD

**Discussion by do-me** [do-me/SemanticFinder#42] [General]
Let's use this discussion for all matters of the Chrome extension.

@VarunNSrivastava it's published and it works! 🥳 

![image](https://github.com/do-me/SemanticFinder/assets/47481567/26c06477-eaba-49a2-9db8-e713670f29b7)

Some thoughts: 
- initial load feels a little buggy and the progress bar flickers quite a bit; I also had to restart Chrome the first time to get it working properly (else it wouldn't find any content)
- we should add some information about the repo on the extension options/info page
- I noticed that sometimes the paragraphs are too long (in comparison to the others), so we should e.g. add a `max chars to split by` option too and/or add some stricter regex. For example it struggles with tables with few delimiters: see my [homepage](https://geo.rocks/about/) and search for anything. 

![image](https://github.com/do-me/SemanticFinder/assets/47481567/cb9b90b7-7000-4853-b61d-c5af4d8dee6c)

I'd just like to underline that you should in no way feel urged to work on any of those points! It's just a list of first UX thoughts that we or (hopefully) any future contributors could work on.

I'll get the social media posts ready and publish tomorrow morning. I held back about requesting yet another Chrome web store review for your recent PR with removed `Tabs` permission as maybe we should bundle the publishing attempts a little - also apparently it's alright with the Tabs permission included.

**Comment by varunneal** [comment]
Woohoo! I've downloaded it and it is in fact working! It would definitely be wonderful to make the extension a bit nicer. Let me know if you make any progress in that regard, otherwise I'll continue making occasional PRs here. Let me know whatever social media/blog posts you make so I can reshare.
